texdoc/__init__.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+"""texdoc extension
+
+This is an extension to search for LaTeX documentation.
+
+Synopsis: <trigger> <query>"""
+
+import os
+import re
+import subprocess
+
+from shutil import which
+from albertv0 import *
+
+__iid__          = "PythonInterface/v0.3"
+__prettyname__   = "TeXdoc"
+__version__      = "1.0"
+__trigger__      = "td"
+__author__       = "Florian Adamsky"
+__dependencies__ = ["texdoc"]
+
+iconPath   = os.path.dirname(__file__)+"/texdoc-logo.svg"
```

Comment:
nit: I'd use `pathlib.Path(__file__)` instead of `os.path`.

Reply:
Wow, TIL! +1 for this, and after playing around in a shell it looks like it supports a fancy way to logically concat parts too.

The only downside is `pathlib.Path(__file__)` returns the file itself, so as far as I can tell we'd need to use the `parent` attribute to get the same result as `os.path.dirname()`: Even *with* that though, it's still more readable than the old method.

```suggestion
iconPath   = Path(__file__).parent / 'texdoc-logo.svg'
```

Reply:
Done! Since iconPath is not a string I need to convert it to a string when I use it. But this is not a problem.

Reply:
@dshoreman haha, glad you liked it :) 

> The only downside is pathlib.Path(__file__) returns the file itself

Yup, exactly, that's why it's always more roboust IMO to use `Path(__file__).absolute().parent`.

> Path(__file__).parent

Hmm, haven't tested it, but does it work? Don't you need to use `.absolute()` before `.parent`?

Reply:
IIRC it returned the full path when called from the basedir *or* the parent dir in my testing with `python ./[path/to/]foo.py`. I also tested with `.resolve().parent` but that didn't seem to affect results. Is there much difference between `.resolve()` and `.absolute()`? I don't recall seeing the latter referenced much.

It might be worth adding `.resolve()` anyway (or absolute?) since the docs suggest it helps with symlinks, but I feel that's likely an edge case so we could probably do it in a separate PR that also updates other plugins to use `Path(...)`